x/twap/README.md

```diff
@@ -6,19 +6,22 @@ A time weighted average price is a function that takes a sequence of `(time, pri
 
 ## Arithmetic mean TWAP
 
-Using the arithmetic mean, the TWAP of a sequence `(t_i, p_i)`, from `t_0` to `t_n`, indexed by time in ascending order, is: $$\frac{1}{t_n - t_0}\sum_{i=0}^{n-1} p_i (t_{i+1} - t_i)$$
+Using the arithmetic mean, the TWAP of a sequence `(t_i, p_i)`, from `t_0` to `t_n`, indexed by time in ascending order, is 
+<p align="center"><img src="https://bit.ly/3cpVdaw" alt="\frac{1}{t_n - t_0}\sum_{i=0}^{n-1} p_i (t_{i+1} - t_i)"/></p>
```

Comment:
Used http://www.sciweavers.org/free-online-latex-equation-editor to generate images